examples/langchain/simple_agent.py

```diff
@@ -0,0 +1,33 @@
+import os
+import mlflow
+
+from langchain.agents import load_tools
+from langchain.agents import initialize_agent
+from langchain.agents import AgentType
+from langchain.llms import OpenAI
+
+assert "OPENAI_API_KEY" in os.environ, "Please set the OPENAI_API_KEY environment variable."
+assert "SERPAPI_API_KEY" in os.environ, "Please set the SERPAPI_API_KEY environment variable."
```

Comment:
cc @vladimirk-db @sueann This example can't be used with Databricks model serving because `SERPAPI` isn't supported. Something for us to think about in the future :)